build/fluent_loader.js

```diff
@@ -20,34 +12,45 @@ module.exports = function(source) {
   if (!locale) {
     throw new Error(`couldn't find locale in: ${this.resourcePath}`);
   }
-  // load default language and "merge" contexts
-  // TODO: make this configurable
-  const en_ftl = fs.readFileSync(
-    require.resolve('../public/locales/en-US/send.ftl'),
-    'utf8'
-  );
-  const en = new FluentBundle('en-US');
-  en.addMessages(en_ftl);
-  // pre-parse the ftl
-  const context = new FluentBundle(locale);
-  context.addMessages(source);
 
-  const merged = merge(en._messages, context._messages);
+  // Parse the current language's translation file.
+  const locResource = FluentResource.fromString(source);
+  let enResource;
+
+  // If the current language is not en-US, also parse en-US to provide a
+  // fallback for missing translations.
+  if (locale !== 'en-US') {
```

Comment:
Ideally, we should work witn an array of languages negotatied by `fluent-langneg`. I found `server/middleware/language.js` but I wasn't sure if changing `req.language` to an array was a good first step. I can look into it a bit more, if you'd like.

Reply:
I created a follow up issue to dig into this, #955. No need to hold up merging this in the meantime :)